MergerService/src/run.cs

```diff
@@ -454,6 +480,19 @@ private void RunTask(MergeTask task, ITaskUtils taskUtils, string? managerCallba
             this._logger.LogDebug($"[{methodName}] end");
         }
 
+        private void UpdateTargetTiles(IData target, List<Tile> tiles, MergeTask task, long overallTileProgressCount, long totalTileCount, ITaskUtils taskUtils)
+        {
+            string methodName = MethodBase.GetCurrentMethod().Name;
+
+            using (this._activitySource.StartActivity("saving tiles"))
+            {
+                this._logger.LogInformation($"[{methodName}] Update {tiles.Count} tiles");
+                target.UpdateTiles(tiles);
+                this._logger.LogDebug($"[{methodName}] UpdateRelativeProgress");
```

Comment:
move this log inside "UpdateRelativeProgress" method

Reply:
done